modules/end-user-guide/partials/proc_creating-a-new-workspace-from-the-dashboard.adoc

```diff
@@ -17,22 +17,27 @@ To edit the devfile:
 
 . In the *Workspaces* window, click the btn:[Add Workspace] button. The *Custom Workspace* page should be opened.
 
-. Scroll down to the *Devfile* section. In the *Devfile editor*, add necessary changes.
+. Scroll down to the *Devfile* section and edit the devfile using *Devfile editor*.
 +
 image::workspaces/minimal-devfile.png[Devfile editor with the minimal devfile, link="../_images/workspaces/minimal-devfile.png"]
 +
-[NOTE]
-.Example: add a project
-====
+For an example demonstration, see below.
```

Comment:
I think "example demonstration" is a tautology. I suggest rewording to smth like "See an example below"